index.html

```diff
@@ -231,7 +231,7 @@ <h2>contact me</h2>
                 Get in Touch
               </button>
               <p class="suc">success</p>
-              <p class="err">Error! Kindly use lower case</p> 
+              <p class="err">Error! Kindly use lower case</p>
```

Comment:
https://github.com/Dinma-Faith/Dinma-Portfolio.github.io/blob/4e0c88ec77f555fe00b1a97b2d8e0f8145bcd4f9/validation.js#L7
- [ ] [REQUIRED] You have done a great job with the form validation @Dinma-Faith , however, there is one thing left:
In your `validate` function, you have a conditional that says if the value of the email is lowerCase, it should display `success` which actually displays but do not forget that the form needs to get submitted normally through `formspree.io`. It is not currently submitting, it only displays success and the previous error message while testing the email field still stays on the screen.
- Kindly ensure that the form gets submitted normally whenever a user submits the contact form with the email field containing lowerCase characters.
- Ensure the error from the UpperCase is not displayed at the same time.

![Screenshot from 2022-04-09 18-39-16](https://user-images.githubusercontent.com/58692590/162585436-1aa01c22-a5c3-4918-b729-00ddc70459cb.png)